src/common/github/jobs/playwright-job.ts

```diff
@@ -0,0 +1,21 @@
+import {Job} from 'projen/lib/github/workflows-model'
+import {job} from './job'
+import {NodeJobOptions} from './node-job-options'
+import {setupNode} from './setup-node'
+
+/**
+ * Create GitHub workflow job which runs a node.js script with a given setup.
+ */
+export const playwrightJob = (options: NodeJobOptions): Job => {
+  const {runsOn, runScriptCommand, workingDirectory} = options
+  const steps = setupNode(options)
+
+  steps.push(
+    {name: 'Install dependencies', run: 'npm install', workingDirectory},
```

Comment:
This won't work with `ppm`. However I don't see why the install step is needed, cause it is already included in the `setupNode`.